src/sknnr_spatial/datasets/_base.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import annotations
+
+from importlib import resources
+
+import numpy as np
+import pandas as pd
+import rasterio
+import rioxarray
+import sknnr.datasets
+import xarray as xr
+from numpy.typing import NDArray
+
+DATA_MODULE = "sknnr_spatial.datasets.data"
+
+
+def _load_rasters_to_dataset(
+    file_names: list[str], *, var_names: list[str], module_name: str, chunks=None
+) -> xr.Dataset:
+    """Load a list of rasters from the data module as an xarray Dataset."""
+    das = []
+    for file_name, var_name in zip(file_names, var_names):
+        bin = resources.open_binary(module_name, file_name)
+
+        da = rioxarray.open_rasterio(bin, chunks=chunks)
+        da = da.to_dataset(dim="band").rename({1: var_name})
+        das.append(da)
+
+    return xr.merge(das)
+
+
+def _load_rasters_to_array(file_names: list[str], *, module_name: str) -> np.ndarray:
```

Comment:
I know you've used `NDArray` as a type hint rather than `np.ndarray` (the return type here).  Is there a reason you went with `np.ndarray` here?

Reply:
Nope, just being inconsistent! Thanks for pointing it out!